session1/3-challenge/5_string.rb

```diff
@@ -5,4 +5,6 @@
 # add_more_ruby("Sad times are ahead!")         # => "Happy times are ahead!"
 
 def add_more_ruby(string)
+	string.gsub("sad", "happy").gsub("Sad", "Happy")
```

Comment:
Hi Nikesh,

If you use regex on the gsub and a hash as a replacement, you could avoid the second gsub operation, like so:
string.gsub(/sad/i, 'sad' => 'happy', 'Sad' => 'Happy')